app/views/issues/evidence/_table.html.erb

```diff
@@ -31,10 +34,22 @@
               when 'Updated'
                 [evidence.updated_at.to_i, local_time_ago(evidence.updated_at)]
               else
-                [evidence.fields.fetch(column, ''), markup(evidence.fields.fetch(column, ''))]
+                liquid_content = evidence.fields.fetch(column, '')
```

Comment:
I don't think `liquid_content` is the best name for this variable since this is the pre-liquid field value